### PR TITLE
chore(ci): Massively reduce # tests we run

### PR DESCRIPTION
**Pull request by timgl**
## Problem

I think we were running a lot of tests twice due to the factory type stuff we were doing

## Changes

get rid of those duplicate tests

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: ee/clickhouse/queries/funnels/test/test_funnel.py [hunk 1]
```diff
@@ -9,18 +8,11 @@
...
 from posthog.queries.funnels.funnel import ClickhouseFunnel
-from posthog.queries.funnels.funnel_persons import ClickhouseFunnelActors
-from posthog.queries.funnels.test.breakdown_cases import funnel_breakdown_test_factory
-from posthog.queries.funnels.test.test_funnel import _create_action, funnel_test_factory
-from posthog.test.base import ClickhouseTestMixin, _create_event, _create_person
+from posthog.test.base import APIBaseTest, ClickhouseTestMixin, _create_person
 from posthog.test.test_journeys import journeys_for
 
 
-class TestFunnelBreakdown(ClickhouseTestMixin, funnel_breakdown_test_factory(ClickhouseFunnel, ClickhouseFunnelActors, _create_event, _create_action, _create_person), funnel_breakdown_group_test_factory(ClickhouseFunnel, ClickhouseFunnelActors, _create_event, _create_action, _create_person)):  # type: ignore
```
Comment (neilkakkar) [inline ee/clickhouse/queries/funnels/test/test_funnel.py:19]: Specific to breakdown factory tests, these are intended to be run multiple times -> What's changing is the type of funnel that's running these tests (some for ordered funnel, some for unordered funnel, etc. etc.)

Comment (neilkakkar) [reply ee/clickhouse/queries/funnels/test/test_funnel.py:19]: Ah, I see, the first factory is duplicated in posthog & ee, while the second one isn't, so just

```
class TestFunnelBreakdown(ClickhouseTestMixin, funnel_breakdown_group_test_factory(ClickhouseFunnel, ClickhouseFunnelActors, _create_event, _create_action, _create_person)): # type: ignore
```

is good here!

**Review by neilkakkar — APPROVED** [review]
Approved assuming comment will be resolved :) 

This makes a lot of sense, we don't need the factory now that there's no postgres based test 🚀

File: ee/clickhouse/queries/funnels/test/test_funnel.py (hunk 1 above)
Comment (timgl) [reply ee/clickhouse/queries/funnels/test/test_funnel.py:19]: Actually surely we should run those against Strict and Unordered too? have added that now.

**Pull request merged**